settings/request_test.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 type ValidSchemeResolver struct {
+	nameUseSupported bool
 }
 
-func (*ValidSchemeResolver) ResolveSpaceIndex(s, i interface{}) (uint32, uint32, error) {
-	var spaceNo, indexNo uint32
+func (*ValidSchemeResolver) ResolveSpace(s interface{}) (uint32, error) {
+	var spaceNo uint32
```

Comment:
Well, we need to have something as methods (so we would not get a panic inside a `newSpaceEncoder` call while trying to get a result of ` res.ResolveSpace(spaceInfo)` and `res.NamesUseSupported()`). But we can just leave
```
func (*ValidSchemeResolver) ResolveSpace(s interface{}) (uint32, error) {
	return 0, nil
}
```
And for tests it will be ok.

Reply:
Ok, nice.

Reply:
Updated.